test_scripts/API/SendLocation/commonSendLocation.lua

```diff
@@ -25,33 +25,36 @@ commonSendLocation.minTimeout = 500
 commonSendLocation.DEFAULT = "Default"
 commonSendLocation.successResultCodes = {
   "SUCCESS",
-  "RETRY",
-  "WARNINGS",
   "SAVED"
 }
 
 commonSendLocation.failureResultCodes = {
-  "UNSUPPORTED_REQUEST",
   "UNSUPPORTED_RESOURCE",
   "DISALLOWED",
   "REJECTED",
   "ABORTED",
-  "IGNORED",
+  "INVALID_DATA",
+  "INVALID_ID",
+  "APPLICATION_NOT_REGISTERED",
+  "OUT_OF_MEMORY",
+  "TOO_MANY_PENDING_REQUESTS",
+  "GENERIC_ERROR",
+  "USER_DISALLOWED"
+}
+
+commonSendLocation.unexpectedResultCodes = {
+  "WARNINGS",
+  "RETRY",
+  "UNSUPPORTED_REQUEST",
   "IN_USE" ,
   "DATA_NOT_AVAILABLE",
   "TIMED_OUT",
-  "INVALID_DATA",
   "CHAR_LIMIT_EXCEEDED",
   "INVALID_ID",
   "DUPLICATE_NAME",
-  "APPLICATION_NOT_REGISTERED",
   "WRONG_LANGUAGE",
-  "OUT_OF_MEMORY",
-  "TOO_MANY_PENDING_REQUESTS",
   "NO_APPS_REGISTERED",
   "NO_DEVICES_CONNECTED",
```

Comment:
@Itileda `"NO_APPS_REGISTERED", "NO_DEVICES_CONNECTED"` should be in `failureResultCodes` table because both of these HMI result codes equal to `APPLICATION_NOT_REGISTERED` mobile result code

Reply:
@AKalinich-Luxoft , fixed in a1dd9de8f0fb3a7642d9a9d9897aff95e763f59e